spec/features/notifications_spec.rb

```diff
@@ -6,7 +6,7 @@
 
     context "from an event" do
       let!(:event)  { create(:event) }
-      it "should create a notification for that event" do
+      pending "should create a notification for that event" do
```

Comment:
it may not look like it, but im fairly certain this is fixed by https://github.com/ReadyResponder/ReadyResponder/pull/325 - I can cherry-pick this to double check.